src/main/java/com/robosh/ejournal/service/SchoolService.java

```diff
@@ -0,0 +1,29 @@
+package com.robosh.ejournal.service;
+
+import com.robosh.ejournal.data.dto.school.SaveSchoolDto;
+import com.robosh.ejournal.data.dto.school.SchoolInfoDto;
+import com.robosh.ejournal.data.entity.School;
+import com.robosh.ejournal.data.mapping.SchoolMapper;
+import com.robosh.ejournal.data.repository.SchoolRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@Slf4j
+@Service
+public class SchoolService {
+
+    private final SchoolRepository schoolRepository;
+
+    private final SchoolMapper schoolMapper;
+
+    public SchoolInfoDto save(SaveSchoolDto saveSchoolDto){
+        School school = schoolMapper.fromSaveSchoolDtoToSchool(saveSchoolDto);
+
```

Comment:
Validate?

Reply:
Is there any custom validation annotation in School?

Reply:
Custom no, but you should process not custom

Reply:
How does validation works without it then?